src/shared/components/FunctionDetailsPanel/components/CodeContainer/domain/useCodeContainer.ts

```diff
@@ -19,9 +19,10 @@ export function useCodeContainer(functionDetails: FunctionDetails): DomainHookRe
     isFetching,
   } = useFetchVCSFile({
     enabled: isLoggedIn,
-    path: functionDetails.fileName,
+    localPath: functionDetails.fileName,
```

Comment:
renamed it to localPath as path started to be imprecise and there are already other places were using `localPath` (for example in the [src/shared/components/GitHubContextProvider/infrastructure/PrivateVcsClient.ts](https://github.com/grafana/explore-profiles/pull/147/files#diff-8af9af8752e9ab90f8fe518712c70117f4104d0aebd78c6c0f0080a0d1265d61))